### ISSUE TITLE
adding a chatBot

### ISSUE THREAD

**Issue by T-Rahul-prabhu-38** [Mohit5Upadhyay/realTimeTrackingApplication#23]
we can add a chatbot so user can ask geographical questions to the chatbot.

@Mohit5Upadhyay  please assign me this task so i can work on it.

**Comment by Mohit5Upadhyay** [comment]
Yes , @T-Rahul-prabhu-38 go ahead to implement chatbot functionality.
